_posts/blog/2019-03-14/2019-03-14-how-to-sell-digital-goods-for-cryptocurrency-on-openbazaar.markdown

```diff
@@ -0,0 +1,69 @@
+
+We would love to recommend a decentralized file hosting solution but the most promising projects are still in an early release stage or pre-release. If you’d like to learn more about projects like [Filecoin](https://filecoin.io/), [Storj](https://storj.io/) and [Sia](https://sia.tech/) you can do so [here](https://coingeek.com/p2p-decentralized-file-storage-dropbox-killer/) and [here](https://medium.com/bitfwd/what-is-decentralised-storage-ipfs-filecoin-sia-storj-swarm-5509e476995f).
+
+You can host files somewhere like your own external website or server, or by using a storage service like Dropbox or Google Drive. Dropbox Professional and Business plans are paid services but they allow you to set a password on files for an extra layer of security around who can access it.
+
+When you receive a payment you can then share a link with your buyer that they can click to download their file. Or, if you want to keep it very simple and your file is small enough you can let buyers know that they need to provide an email address so you can email the file to them when their payment is confirmed.
+
+#### 2. Create your listing
+
+Now that you have a plan for hosting and sharing your digital product you can create a listing for it on OpenBazaar. Create a **new listing** and under **Type** select **“Digital Good.”**
+
+Give it a great **title** and **descriptio**n that relates to keywords people may use in search to find it easily. Make sure to also describe how you will send the file to users when they complete their purchase so they know what to expect and where, whether in OpenBazaar chat or via email. It’s also helpful to include any instructions they may need for setting up or installing your file if it requires that.
+
+Include at least 1 great **image** that describes your item such as a book jacket mockup for an ebook or the cover art for a music album. 
+
+Leave the **shipping** option blank because you don’t need to worry about shipping! Hooray!
+
+**Tags** can be added to boost the searchability of your item. Include obvious ones that relate to the item itself like “book” or “music” then more as relates to broader categories it represents such as “fiction” or “bluegrass.”
+
+A **category** is to help keep your store organized. If you plan to sell both ebooks AND music albums, you’d just type in the corresponding category for each of your listings.
+
+**Variants** and **inventory** can likely be left blank as well. These are helpful for things like physical goods where you may offer the same hat but in 3 different colors with only a certian number of each available. If you have variations with your digital goods it may be worth creating separate listings for each.
```

Comment:
Remove "as well" since the shipping comment it is referencing should be removed.

Reply:
"variations of"